src/extensions/surveys/survey.css

```diff
@@ -285,7 +286,14 @@
         gap: 8px;
         font-size: 13px;
         align-items: center;
-        flex-wrap: wrap;
+
+        & span {
+            color: inherit;
+        }
+
+        &.choice-option-open {
+            flex-wrap: wrap;
+        }
```

Comment:
and this fixes the line-break problem, by only wrapping if it's an open-ended input